docs/redoc/master/openapi.json

```diff
@@ -5724,15 +5724,11 @@
         ]
       },
       "MultiVectorConfig": {
-        "anyOf": [
-          {
-            "$ref": "#/components/schemas/MaxSimConfig"
-          }
+        "type": "string",
+        "enum": [
+          "max_sim"
         ]
```

Comment:
I currently don't know, but is this still compatible with `anyOf`? such that if we add more variants -which can be objects- can still be represented fine? Off the top of my mind I can't remember if we have something like this already

Reply:
I don't think we could have other non string type in the enum with this encoding.

Reply:
after thinking about your comment, I will push a different encoding to make sure we can add new fields to the `MultiVectorConfiguration` :)

Reply:
I like the new approach, analog to the `distance` param